bus-mapping/src/circuit_input_builder/transaction.rs

```diff
@@ -343,6 +343,7 @@ impl Transaction {
         );
 
         let tx_type = TxType::get_tx_type(eth_tx);
+        log::info!("bus mapping: tx_type {:?}", tx_type);
```

Comment:
change to debug or trace?    "bus mapping" , the module name will be in log line, not needed here.   You can change it like log::debug!("{}th tx, tx hash {}, tx_type {}").

Reply:
changed to  log::debug! in [93967d5](https://github.com/scroll-tech/zkevm-circuits/pull/1412/commits/93967d5fec533fbf2a1b573b87b7dcca4fa38f58)